libssh2-sys/Cargo.toml

```
@@ -1,6 +1,6 @@
 [package]
 name = "libssh2-sys"
-version = "0.2.23"
+version = "0.2.24"
```

My understanding is that if we leave the minor version at `0.2` then the assumption is that code that used `LIBSSH2_USERAUTH_KBDINT_PROMPT` in that version will continue to compile when `cargo update` or similar upgrades them to this newer version.  I'm not sure if that is true because the sign of one field has changed and potentially the size of the whole struct (`size_t` is a bit wacky!).

I think that means that we need to bump the minor version number to uphold semver rules.  However, I remember @alexcrichton wanting to resist that in the past, for the sake of ease-of-upgrading `cargo` itself.

```suggestion
version = "0.3.0"
```

---

That is true. I will wait for @alexcrichton's call on which way to go.

---

@alexcrichton Just to confirm, you are O.K. with us making a bump to `0.3.0`, correct?

---

Well to clarify I don't work on Cargo any more, and this crate I haven't worked on in ages (I should really transfer it...), so whether or not I'm ok with a bump I don't think should have much effect on the decision because I won't be the one dealing with the fallout.